release/changes.md

```diff
@@ -1 +1 @@
-
+* Fix ```pushd``` ordering in ```invoke_gradle```
```

Comment:
This will end up in the release notes for the release, so we should make sure this is something someone using the scripts would understand. As such, I suggest rewording this to describe the bug that this change fixes (something about how specifying a project directory was broken when running the Gradle experiments).

Reply:
Sure thing, I updated the comment!